packages/ui/src/views/platform/entity-reference-component.tsx

```diff
@@ -42,12 +42,35 @@ export interface EntityReferenceProps<T extends BaseEntityProps, S = string, F =
   // Search
   searchValue?: string
   handleChangeSearchValue: (val: string) => void
+
+  // Icons for default entity renderer
+  icon?: IconPropsV2['name']
+
+  // Custom entity comparison
+  isEntityEqual?: (entity1: T, entity2: T) => boolean
```

Comment:
let's call this `compareFn` to be consistent with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#comparefn